config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,10 @@
+CarrierWave.configure do |config|
+  if !Rails.env.test?
+    config.fog_credentials = {
+      provider: "AWS",
+      aws_access_key_id: ENV["AWS_ACCESS_KEY_ID"],
+      aws_secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"]
+    }
+    config.fog_directory  = ENV["S3_BUCKET"]
```

Comment:
Unnecessary spacing detected.<br>Operator `=` should be surrounded by a single space.